api/osb/check_instance_visibility_plugin.go

```diff
@@ -43,10 +43,11 @@ func (p *checkVisibilityPlugin) Provision(req *web.Request, next web.Handler) (*
 	if err := decodeRequestBody(req, requestPayload); err != nil {
 		return nil, err
 	}
-	planID, err := findServicePlanIDByCatalogIDs(ctx, p.repository, requestPayload.BrokerID, requestPayload.ServiceID, requestPayload.PlanID)
+	plan, err := findServicePlanIDByCatalogIDs(ctx, p.repository, requestPayload.BrokerID, requestPayload.ServiceID, requestPayload.PlanID)
```

Comment:
findServicePlanIDByCatalogIDs -> findServicePlanByCatalogIDs (the function no longer finds an ID)